nextcord/ext/help_commands/embedded.py

```diff
@@ -0,0 +1,97 @@
+from typing import TYPE_CHECKING
+
+from nextcord.ext.commands import Cog, Command, Group, HelpCommand
+
+from nextcord import Embed, Message
+
+if TYPE_CHECKING:
```

Comment:
As said above, why do you need this? The `typing` module won't cause any circular imports at all.